appengine/endpoints/app.rb

```diff
@@ -20,11 +20,21 @@
 require "base64"
 require "json"
 require "sinatra"
+require "rack/cors"
 
 before do
   content_type :json
 end
 
+use Rack::Cors do
```

Comment:
Aja pointed me to: https://til.hashrocket.com/posts/4d7f12b213-rails-5-api-and-cors